common/gas_limits.go

```diff
@@ -0,0 +1,7 @@
+package common
+
+const (
+	EVMSend = 21000
+	// Todo Move gas limits from zeta-client to this file
```

Comment:
```suggestion
	// TODO: Move gas limits from zeta-client to this file
```
Just so that we can ctrl+f